### ISSUE TITLE
conversion of any article in taxpub

### ISSUE THREAD

**Issue by myrmoteras** [plazi/ggxml2taxpub-treatments#52]
@gsautter is this useful for any article or just EJT?  

https://tb.plazi.org/GgServer/xslt/534F336D7706BD1BFD89FB90B945539D?xsltUrl=https://raw.githubusercontent.com/plazi/ggxml2taxpub-treatments/main/xslt/gg2tp_l1.xsl 

does this produce valid taxpub?

**Comment by gsautter** [comment]
This should work for every treatment from every article ... it's actually what we use to generate the TaxPub we export to SIB, which is validated before export.

The same TaxPub version of treatments is also available at https://tb.plazi.org/GgServer/taxPubL1/534F336D7706BD1BFD89FB90B945539D

**Comment by myrmoteras** [comment]
@gsautter this is not about treatments, but entire articles, as we tried to achieve with EJT. 
Any options there?

**Comment by gsautter** [comment]
Well, this XSLT is for treatments, and your URL ran it on a treatment ... sorry about the confusion.

I don't think this XSLT will do at the trick article level, simply because it starts from the `treatment` element ... no abstract, no bibliography, no figures or tables ...

@tcatapano is working on an XSLT for whole articles ...

**Comment by myrmoteras** [comment]
Didn't we have a start for the EJT articles?  https://github.com/plazi/EJT-testbed/tree/master/valid_taxpub

**Comment by gsautter** [comment]
Yes, and we still have it, and still have it running and putting EJTs on GitHub as TaxPub ... but AFAIK it's a bit too detailed for SIB's needs, and too prone to lose validity at that very detail level ...

@tcatapano is using it a starting point for aforementioned article level XSLT, but I don't know anything about progress in this regard.
